mindbender/tools/projectmanager/app.py

```diff
@@ -0,0 +1,230 @@
+import sys
+
+from ... import io, schema
+from .. import lib as parentlib
+from ...vendor.Qt import QtWidgets, QtCore, QtGui
+from ...widgets import assetwidget
+
+from .dialogs import TasksCreateDialog, AssetCreateDialog
+from .model import TasksModel
+
+module = sys.modules[__name__]
+module.window = None
+
+
+class Window(QtWidgets.QDialog):
+    """Project manager interface
+
+    """
+
+    def __init__(self, parent=None):
+        super(Window, self).__init__(parent)
+        project_name = io.active_project()
+        self.setWindowTitle("Project Manager ({0})".format(project_name))
+        self.setFocusPolicy(QtCore.Qt.StrongFocus)
+        self.setAttribute(QtCore.Qt.WA_DeleteOnClose)
+
+        # assets
+        assets_widgets = QtWidgets.QWidget()
+        assets_widgets.setContentsMargins(0, 0, 0, 0)
+        assets_layout = QtWidgets.QVBoxLayout(assets_widgets)
+        assets = assetwidget.AssetWidget()
+        assets.view.setSelectionMode(assets.view.ExtendedSelection)
+        add_asset = QtWidgets.QPushButton("Add asset")
+        assets_layout.addWidget(assets)
+        assets_layout.addWidget(add_asset)
+
+        # tasks
+        tasks_widgets = QtWidgets.QWidget()
+        tasks_widgets.setContentsMargins(0, 0, 0, 0)
+        tasks_layout = QtWidgets.QVBoxLayout(tasks_widgets)
+        label = QtWidgets.QLabel("Tasks")
+        label.setFixedHeight(28)
+        task_view = QtWidgets.QTreeView()
+        task_view.setIndentation(0)
+        task_model = TasksModel()
+        task_view.setModel(task_model)
+        add_task = QtWidgets.QPushButton("Add task")
+        tasks_layout.addWidget(label)
+        tasks_layout.addWidget(task_view)
+        tasks_layout.addWidget(add_task)
+
+        body = QtWidgets.QSplitter()
+        body.setContentsMargins(0, 0, 0, 0)
+        body.setSizePolicy(QtGui.QSizePolicy.Expanding,
```

Comment:
It shouldn't matter, it should have gotten sifted out of PySide.

https://github.com/mottosso/Qt.py/blob/52e3c073c1baeef2c871a6e546a93458e84590a8/Qt.py#L314

But I see what's going on, there's an older version in this project. I'll update this so it can't happen again.